pkg/kubelet/kubelet.go

```diff
@@ -1045,7 +1051,6 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 			}
 		}
 	}
-	containersToKeep[podInfraContainerID] = empty{}
```

Comment:
Ah yes I see it now. Could we move it up to here instead? It seems to follow better here.